### DISCUSSION TITLE
Add IPv6 Support

### DISCUSSION THREAD

**Discussion by LordSimal** [spatie/ssl-certificate#173] [Ideas]
I would love to be able to call this functionality 
```
$certificate = SslCertificate::download()
    ->fromIpAddress($ipAddress)
    ->forHost($hostName);
```
with a valid IPv6 address as well.

But currently the `->fromIpAddress()` method calls `filter_var($ipAddress, FILTER_VALIDATE_IP)` which only works on IPv4 addresses without any further options.

**Comment by freekmurze** [comment]
Feel free to create a PR for this. Make sure to include tests.

**Comment by LordSimal** [comment]
Do you have any recommendations for which IPv6 address (or rather which domain) I use to test the functionality?
spatie.be doesn't seem to have a IPv6 address.
Otherwise I would choose something like google.com

**Reply by freekmurze** [reply]
google.com is fine 👍
